src/encoded/schemas/tissue.json

```diff
@@ -113,7 +127,7 @@
         },
         "recovery_interval": {
```

Comment:
If this property is going to remain and a new property of the same name is added to tissue_collection with a different meaning then this could cause confusion.  I don't think this will be a problem for indexing per se as items are namespaced (may want to confirm with @willronchetti ) but general searches could be confounded.  If the goal is to remove this from tissue after the new prop is added to tissue_collection then no worries but consider renaming one or the other?

Reply:
Yes, my plan was to remove this property from Tissue once I had transferred the values to a new TissueCollection, unless this people think this is inadvisable.